README.md

```diff
@@ -17,7 +17,8 @@ List of projects bundling Minio server in it's stack.
 
 ### Awesome Use Cases
 List of projects using Minio server to store their data.
-* [www.houbank.com](https://www.houbank.com) - www.houbank.com uses Minio as a simple, reliable and compatible self-host AWS S3 alternative. Houbank uses Minio to upload user files and exchange documents in internal applications/jobs.
+* [Apcera](https://www.apcera.com/)- Apcera is a container management platform for cloud-native and legacy applications. Apcera uses Minio as its backend object storage server.
```

Comment:
They are providing minio as a service,  i don't know if we can add them as using Minio as backend.